### PR TITLE
IDEM-89

### PR DESCRIPTION
**Pull request by JasminFarhi**
Image from iOS (112)

### CODE REVIEW

File: app/src/context/Exchange.tsx [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+      if (initialGpibUserID) {
+        setGpibUserID(initialGpibUserID.gpibUserID);
+      }
+    })();
+  }, []);
+
+  const shareDetailsAlert = () => {
+    Alert.alert(
+      "Share Details",
+      `Sign up successful, your temporary password is ` + randomTempPassword,
```
Comment (bitcoinbrisbane) [inline app/src/context/Exchange.tsx:39]:
```suggestion
      `Sign up successful, your temporary password is ${randomTempPassword}`,
```

**Review by bitcoinbrisbane — DISMISSED** [review]
One small thing, and SonarQube

**Review by bitcoinbrisbane — APPROVED** [review]

File: app/src/components/RequestClaimsModal.tsx [hunk 2]
```diff
@@ -55,6 +55,7 @@ const RequestClaimsModal: React.FC<Props> = ({ claimRequest, onClose }) => {
         console.error(err?.response?.data || error);
       }
     };
+    //USE THIS TO POST THE CLAIM TO GPIB API
```
Comment (jrocca82) [inline app/src/components/RequestClaimsModal.tsx:58]: ?

**Review by jrocca82 — APPROVED** [review]

File: app/src/navigation/screens/VendorDetails.tsx [hunk 3]
```diff
@@ -4,12 +4,17 @@ import { View, Image, StyleSheet, Text, Dimensions } from "react-native";
...
+import { useExchange } from "../../context/Exchange";
+import { useClaimValue } from "../../context/ClaimsStore";
 
 const VendorDetailsScreen: React.FC = () => {
   const { vendors } = useVendorsList();
+  const { makeGpibUser } = useExchange();
   const route = useRoute<VendorStackNavigationRoute<"VendorDetails">>();
-
+  const name = useClaimValue("FullNameCredential");
+  console.log(name, "name");
```
Comment (jrocca82) [inline app/src/navigation/screens/VendorDetails.tsx:15]: remove

File: app/src/navigation/screens/VendorDetails.tsx [hunk 4]
```diff
@@ -4,12 +4,17 @@ import { View, Image, StyleSheet, Text, Dimensions } from "react-native";
...
+  const email = useClaimValue("EmailCredential");
+  console.log(email, "email");
```
Comment (jrocca82) [inline app/src/navigation/screens/VendorDetails.tsx:17]: remove

**Review by jrocca82 — APPROVED** [review]

**Review by bitcoinbrisbane — APPROVED** [review]

**Pull request merged**